src/transformState.js

```diff
@@ -1,11 +1,46 @@
 'use strict';
 
-/**
- * @param {Object} state
- * @param {Object[]} actions
- */
+// /**
+//  * @param {Object} state
+//  * @param {Object[]} actions
+//  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const dataToAdd = action.extraData;
+    const keysToRemove = action.keysToRemove;
```

Comment:
```suggestion
    const { type, extraDate, keysToRemove} = action
```
better to destructure action